### PR TITLE
Step 7: Dynamic portfolio section

### PR DESCRIPTION
**Pull request by leehaney254**
## In this section we: 👍 
- created a dynamic portfolio section
- created the cards of the project dynamically
- created a pop-up window dynamically

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -63,4 +63,268 @@ function fillData() {
...
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
```
Comment (tufoinnkuo10) [inline app.js:80]: - [x] Great job. please centralize your mobile version image so it looks like the figma version
- [ ] kindly adjust the tech buttons as well. also, ensure the see source and see live buttons are of same size. the button icons well adjusted as well. great job

yours
Screenshot (604)_LI
figma
Screenshot (600)

File: app.js [hunk 2]
```diff
@@ -63,4 +63,268 @@ function fillData() {
...
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './assets/rest-card.png',
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
```
Comment (tufoinnkuo10) [inline app.js:88]: - [x] please ensure that you follow the figma design as much as possible. resize and position your image in the desktop popup so it takes the figma design.

your design
Screenshot (605)_LI
figma
Screenshot (599)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @leehaney254 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by leehaney254** [comment]
Hello!
- I have resized and positioned my desktop image to follow the figma design. 💪 
- I have centralized my image to match that of figma.
- I have adjusted the tech buttons
- Fixed the source and live button 👍

File: app.js [hunk 3]
```diff
@@ -63,4 +63,274 @@ function fillData() {
...
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
```
Comment (Qoosim) [inline app.js:79]: - [ ] Good job in implementing the suggestions from the previous reviewer. However, there is still one issue you need to fix on this part of the project. Some content of the page cut off at the bottom of your modal on the mobile. Kindly look into it and fix it.

### Your Design
mobile-modal

File: app.js [hunk 4]
```diff
@@ -63,4 +63,274 @@ function fillData() {
...
+  {
+    name: 'Profesional Art Printing Data',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    image: './assets/rest-card.png',
+    desktopImg: './assets/Img-Placeholder.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
```
Comment (Qoosim) [inline app.js:87]: - [ ] The same issue affects the desktop version. Part of the modal is cut off at the bottom of the popup. Kindly look into it and fix it.

### Your Design
desktop-modal

**Review by Qoosim — CHANGES_REQUESTED** [review]
#   *Status: Required Changes* :recycle: 

Hi @leehaney254, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leehaney254** [comment]
I have fixed the breaking code on mobile version 💪

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @leehaney254, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

- [ ] **[OPTIONAL]:** I noticed that your project has a horizontal overflow, I would suggest you look into it and fix it. Having a good-looking UI design will make sure future recruiters get more attracted to your work! 🤓 🛠️ 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**